Gemfile

```diff
@@ -59,6 +59,7 @@ group :system_tests do
   gem 'rbnacl', '>= 4',                     :require => false
   gem 'rbnacl-libsodium',                   :require => false
   gem 'bcrypt_pbkdf',                       :require => false
+  gem 'ed25519',                       :require => false
```

Comment:
This file is managed by modulesync, so you'll need to add this gem to `.sync.yml` (and could you also fix the indentation).
See https://github.com/voxpupuli/puppet-redis/blob/d0ee587d0d0327ef85d5701fbd6c9d9f33569fbe/.sync.yml#L10-L14 for an example.

Reply:
Thanks, slipped it while doing local testing. I can't easily run acceptance tests for any module locally without that gem so I'll send a PR to add to: https://github.com/voxpupuli/modulesync_config/blob/master/config_defaults.yml